src/htcondorce/info_query.py

```diff
@@ -141,14 +139,14 @@ def filterResourceAds(constraints, resources):
 
     """
     predicates = []
-    for attr, value in constraints.iteritems():
+    for attr, value in iter(constraints.items()):
```

Comment:
```suggestion
    for attr, value in constraints.items():
```
items() already returns an iterator.